src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FlexContainer } from '../FlexContainer';
+import { FilterRow } from './FilterRow';
+import { FilterProps } from './Filters.types';
+import { OptionPropType } from './Select/Select.types';
+
+const Filters: React.FC<FilterProps> = ({ options }) => (
+  <FlexContainer flexDirection="column">
+    {options.map(({ uuid, ...rest }, index) => (
+      <FilterRow key={uuid} isFirstRow={index === 0} uuid={uuid} {...rest} />
+    ))}
+  </FlexContainer>
+);
+
+export default Filters;
+
+const RowPropType = PropTypes.exact({
```

Comment:
Will this be reused somewhere? If yes please move it to `Filters.types.ts` if not put it directly in `Filters.propTypes`